cli/ops/host.py

```diff
@@ -17,11 +17,16 @@ def check_host_status(node, hostname, status=None):
         status: custom status check for the host
     """
     conf = {"host_pattern": hostname, "format": "json-pretty"}
-    out = loads(CephAdm(node).ceph.orch.host.ls(**conf))
+    out = loads((CephAdm(node).ceph.orch.host.ls(**conf))[node[0].hostname][0])
```

Comment:
Hi @sayaleeraut 

Can you please not change lib for specific hosts since same lib used in other suites as well (Impact other suites as well)

If required to use method with changes , Please create in test file

Reply:
Yes, done, please review @MohitBis